ta/os_test/os_test.c

```diff
@@ -988,6 +988,8 @@ TEE_Result ta_entry_bad_mem_access(uint32_t param_types, TEE_Param params[4])
 	long int stack = 0;
 	long int stack_addr = (long int)&stack;
 	void (*volatile null_fn_ptr)(void) = NULL;
+	char *zero_size_malloc = NULL;
+	volatile char c = 0;
```

Comment:
why `volatile`?

Reply:
Because otherwise the read `c = *zero_size_malloc;` is optimized out by the compiler... (the value is not used)